src/romein_origin.cu

```diff
@@ -0,0 +1,641 @@
+/*
```

Comment:
What is this file?

Reply:
I just realize that this repo is not updated. Let me update a couple of files and remove them before we go ahead with this pull request. This is the original romein kernel, but we could remove this and keep romein.cu which is just a clean up version of the original romein